src/app/layoutsite/layoutsite.component.ts

```diff
@@ -1,17 +1,45 @@
-import {Component} from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 import {MatSlideToggleChange} from "@angular/material/slide-toggle";
 
 @Component({
   selector: 'app-layoutsite',
   templateUrl: './layoutsite.component.html',
   styleUrls: ['./layoutsite.component.scss'],
 })
-export class LayoutSiteComponent {
+export class LayoutSiteComponent implements OnInit{
+  onOff : boolean = false;
+  ngOnInit() {
+    this.checkOnOff();
+  }
+
+  checkOnOff() {
+    const header = document.getElementsByTagName("mat-toolbar")[0];
```

Comment:
Don't use this in angular instead use ElementRef via @ViewChild https://angular.io/api/core/ViewChild